Experimental/conda/readme.md

```diff
@@ -0,0 +1,25 @@
+Create a conda environment for running fab
+```
+conda env create -f environment.yml
+```
+
+
+Activate the new environment
+
+```
+conda activate fab
+```
+
+Install fab (from the fab folder)
+```
+pip install .
+```
+
+[Editable install](https://pip.pypa.io/en/stable/cli/pip_install/#editable-installs) for developers
+```
+pip install -e .
+```
+
+Please be aware of some considerations when
+[using pip and conda](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#using-pip-in-an-environment)
+together.
```

Comment:
We could perhaps provide two alternate methods for the installation of `fab` itself?  I wasn't aware of the steps here and usually I do `python setup.py install` or `python setup.py develop` (for the equivalent of an "editable install") - So that would be "Install Fab (Pip)" or "Install Fab (setuptools)" or something like that?

Might also be worth a mention of how to un-install Fab from the environment (as could be needed if/when switching between working copies or similar) - for my method that would be `python setup.py install --uninstall`  (or `develop --uninstall`)